--- /usr/src/redhat/SPECS/bash.spec 2013-06-27 02:43:12.000000000 -0600
+++ bash.spec   2014-02-18 11:01:49.000000000 -0700
@@ -104,6 +104,10 @@
 # 978840, fix deadlock in trap, backported from devel branch
 Patch155: bash-3.2-bz964753.patch

+# A patch for syslog for bash 3.2 found on github
+Patch156: bash-3.2-syslog.patch
+
+
 Requires: mktemp
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)

@@ -191,6 +195,8 @@

 %patch155 -p1 -b .bz964753

+%patch156 -p1 -b .syslog.patch
+
 echo %{version} > _distribution
 echo %{release} > _patchlevel

@@ -331,6 +337,10 @@
 %doc doc/*.ps doc/*.0 doc/*.html doc/article.txt

 %changelog
+* Tue Feb 18 2014 Seth Miller <seth@migrantgeek.com> - 3.2-24
+- Added bash-3.2-syslog-patch
+  Patch will log all commands to LOCAL5 with the username and PID
+
 * Thu Jun 27 2013 Roman Rakus <rrakus@redhat.com> - 3.2-32.1
 - Fixed a bug that caused trap handlers to be executed recursively,
   corrupting internal data structures.
